OpenEdXMobile/res/layout/fragment_course_unit_grade.xml

```diff
@@ -5,17 +5,80 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@color/white"
-        android:gravity="center"
         android:orientation="vertical">
 
-        <org.edx.mobile.view.custom.IconImageViewXml
-            android:id="@+id/content_error_icon"
-            android:layout_width="@dimen/unit_unavailable_error_icon_size"
-            android:layout_height="@dimen/unit_unavailable_error_icon_size"
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
This layout can use some auto layout to fix some indentations in it.

Reply:
done